terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/ApServerName.java

```diff
@@ -0,0 +1,12 @@
+package org.terasoluna.gfw.functionaltest.app;
+
+/**
+ * Enumeration class for identifying application server.
+ * <p>
+ * If application server name is not set in application.env.properties, <br/>

```

Comment:
I did `mvn formatter:format` and commit at `Applied the mechanism` changed file at a3b867f28e443d48588e823764958b1136b1d5c2 commit and `Change assertion using the mechanism` changed file at